pages/common/blender.md

```diff
@@ -6,27 +6,27 @@
 
```

Comment:
I think we can update the more information link on this page to https://docs.blender.org/manual/en/latest/advanced/command_line/arguments.html in a future PR.

Reply:
Hmm... New idea - try fix all broken links via script. Use manned as a fallback if smth can be found by our request.